assets/js/util/partition-report.js

```diff
@@ -22,9 +22,10 @@
 import invariant from 'invariant';
 
 /**
- * Splits report data into segments of equal length for current and comparison ranges.
+ * Splits report data into segments for current and comparison ranges.
  *
  * @since 1.33.0
+ * @since n.e.x.t Updated to support incomplete `report` arrays.
```

Comment:
In case we want to include this in the upcoming release:
```suggestion
 * @since 1.34.0 Updated to support incomplete `report` arrays.
```